client/sourceeditor/codemirror.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html dir='ltr'>
+  <head>
+    <style>
+      html, body { height: 100%; }
+      body { margin: 0; overflow: hidden; }
+      .CodeMirror { width: 100%; height: 100% !important; line-height: 1.25 !important;}
+    </style>
+    <link rel="stylesheet" type="text/css" href="/client/themes/common.css" />
```

Comment:
Good idea, using a separate page makes resolving these paths much easier.  Can you please make all these paths relative to the html page instead? so `/client/sourceeditor/codemirror/` -> `codemirror/` and `/client/shared/theme-switching.js` -> `../shared/theme-switching.js`.